src/components/directory-content.js

```diff
@@ -357,6 +358,8 @@ const DirectoryContent = () => {
             } else if (objectSubtype === FilterType.FORM) {
                 return <FilterListIcon className={classes.icon} />;
             }
+        } else if (objectType === ElementType.CASE) {
+            return <PhotoIcon />;
```

Comment:
Add `className` as well here for icon size and all...

Reply:
done